manifest.json

```diff
@@ -4133,6 +4133,17 @@
                 }
             },
             "versions": {
+                "1.4.0": {
+                    "changelog": "Add a config to enable friend links on the notifications - EIA485",
+                    "releaseUrl": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/tag/v1.3.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/download/v1.3.0/UserJoinLeaveNotifications.dll",
```

Comment:
looks like the reason for the checksums not matching is this link being incorrect

Reply:
https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/download/v1.4.0/UserJoinLeaveNotifications.dll returns a file with the correct checksum